server/src/main/scala/net/wiringbits/repositories/daos/UserNotificationsDAO.scala

```diff
@@ -0,0 +1,64 @@
+package net.wiringbits.repositories.daos
+
+import net.wiringbits.repositories.models.{NotificationStatus, UserNotification}
+
+import java.sql.Connection
+import java.time.Instant
+import java.util.UUID
+
+object UserNotificationsDAO {
+
+  import anorm._
+
+  def create(request: UserNotification.Create)(implicit conn: Connection): Unit = {
+    val _ = SQL"""
+      INSERT INTO user_notifications
+        (user_notification_id, user_id, notification_type, subject, message, status, execute_at)
+      VALUES (
+        ${request.id.toString}::UUID,
+        ${request.userId.toString}::UUID,
+        ${request.notificationType.toString}::TEXT,
+        ${request.subject},
+        ${request.message},
+        ${request.status.toString}::TEXT,
+        ${request.executeAt}::TIMESTAMPTZ
+      )
+      """
+      .execute()
+  }
+
+  def getPendingNotifications()(implicit conn: Connection): List[UserNotification] = {
+    SQL"""
+      SELECT user_notification_id, user_id, notification_type, subject, message, status, status_details, error_count, execute_at, created_at, updated_at
+      FROM user_notifications
+      WHERE status != ${NotificationStatus.Success.toString}
+        AND execute_at <= NOW()
```

Comment:
Let's prefer `clock.instant` over `NOW()`.

We could define the max errors where we won't try anymore, possibly 10 is good enough.